lib/network/modules/Clustering.js

```diff
@@ -726,6 +736,8 @@ class ClusterEngine {
   * Using a base edgeId, update all related clustered edges with the new options
   * @param startEdgeId
   * @param {object} newOptions
+  *
+  * NOTE: never ever called!
```

Comment:
Ah, OK. Good catch. Didn't grep on those.

I only see it in one example, though: `network/other/changingClusteredEdgeNodes()`. Are there more?